modules/dailymotionBidAdapter.js

```diff
@@ -42,11 +42,15 @@ function getVideoMetadata(bidRequest, bidderRequest) {
     iabcat2: Array.isArray(videoParams.iabcat2)
       ? videoParams.iabcat2
       : Object.keys(iabcat2),
-    id: videoParams.id || '',
-    lang: videoParams.lang || '',
+    id: videoParams.id ||
+      deepAccess(bidderRequest, 'ortb2.site.content.id', ''),
```

Comment:
For the context, is this will be used by non-browser app ? (like Dailymotion mobile app?)
If this is the case, the path could be `ortb2.site.content.id` instead of `ortb2.app.content.id`

Reply:
That's a good find, I wonder if it's applicable as we only support `video` adUnits (and not `native`).
cc: @sebmil-daily